changelog/_unreleased/2020-09-17-fix-product-media-position-duplication.md

```diff
@@ -0,0 +1,34 @@
+---
+title:         Fix bug when images are add to a product the position 1 can be assigned multiple times
+issue:         NEXT-9292
+author:        Joshua Behrens
+author_email:  code@joshua-behrens.de
+author_github: JoshuaBehrens
+___
```

Comment:
Also the template is not right. The meta information have to be wrapped by `---` and not `___`.

Reply:
I copied it strait from the _example.md

<img width="222" alt="grafik" src="https://user-images.githubusercontent.com/1133593/93466804-14e96880-f8ed-11ea-905f-1806bc8fa630.png">

It contains the underscores as well.

Reply:
Shall I fix it as well? :)

Reply:
Nvm I deleted the wrong separator 🥴